jekyll/_cci2_ja/about-circleci.md

```diff
@@ -23,7 +23,7 @@ Linux、macOS、Android の各プラットフォームに加え、SaaS やオン
 
 GitHub もしくは Bitbucket アカウントの認証が完了し、各リポジトリ内のプロジェクトが [circleci.com](https://circleci.com) に追加されると、その後はコードに変更があるたびに、まっさらな状態のコンテナや VM 上で自動的にテストが実行されます。CircleCI は、個別の[コンテナ]({{site.baseurl}}/ja/2.0/glossary/#container)または VM でそれぞれの[ジョブ]({{site.baseurl}}/ja/2.0/glossary/#job)を実行します。つまりジョブを実行するたびに、CircleCI はジョブを実行するためのコンテナまたは VM をスピンアップします。
 
-テスト完了後にはメールで成功・失敗の通知が届くほか、Slack、HipChat、Campfire、Flowdock、IRC などのチャットツールと連携して通知を受け取ることも可能です。テスト通知の内容は、レポートライブラリが追加されているプロジェクトであれば、その詳細ページから確認できます。
+テスト完了後にはメールで成功・失敗の通知が届くほか、Slack、IRC などのチャットツールと連携して通知を受け取ることも可能です。テスト通知の内容は、レポートライブラリが追加されているプロジェクトであれば、その詳細ページから確認できます。
```

Comment:
Unfortunately I don't know the correct Japanese grammar for a two-item list... May need additional review.

Reply:
@michelle-luna / @teesloane just wondering if it would be best to leave this edit out as I think the translators get notified of changes to the english language content automatically?